404.html

```diff
@@ -1,241 +1,255 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link
+			rel="shortcut icon"
+			href="assets/logos/favicon.svg"
+			type="image/svg+xml"
+		/>
+		<title>If These Lands Could Talk - 404 Page Not Found</title>
+		<!-- google fonts -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<!-- styles -->
+		<link rel="stylesheet" href="css/global.css" />
+		<link rel="stylesheet" href="css/404.css" />
+		<!-- MailerLite Universal -->
+		<script src="/js/mailerlite.js" defer></script>
+	</head>
+
+	<body>
+		<!-- Header Navigation-->
+		<header>
+			<!-- Navigation -->
+			<nav>
+				<div class="wrapper">
+					<div class="flex-container">
+						<!-- ITLCT Logo -->
+						<a
+							href="/index.html"
+							class="img-link"
+							aria-label="If These Lands Could Talk Home Page Link"
+						>
+							<img
+								src="/assets/logos/itlct-logo.svg"
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+						<!-- Navigation Menu -->
+						<div class="mobile-menu-container">
+							<!-- mobile menu button -->
+							<button
+								id="mobile-menu"
+								class="mobile-menu"
+								aria-label="toggle mobile menu on"
+							>
+								<!-- Hamburger Icon -->
+								<img
+									src="/assets/icons/global/menu-icon.svg"
+									alt="mobile menu icon"
+									class="menu-icon"
+								/>
+								<!-- exit menu icon -->
+								<img
+									src="/assets/icons/global/close-icon.svg"
+									alt="exit mobile menu icon"
+									class="menu-exit-icon hidden"
+								/>
+							</button>
+
+							<!-- Page Links -->
+							<ul id="nav-links" class="nav-links">
+								<li><a href="/about.html">About Us</a></li>
+								<li><a href="/impact.html">Our Impact</a></li>
+								<li><a href="/values.html">Our Values</a></li>
+								<li><a href="/events.html">Events</a></li>
+								<li><a href="/team.html">Our Team</a></li>
+								<!-- Mobile Contact Button -->
+								<li class="mobile-support-btn">
```

Comment:
FYI: contact link has been removed from the nav since the new CTA link right beside it. Contact us is currently taking place of support us links

Desktop view:
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/a7e9bff1-aefe-42d0-8886-ce7283db5385)

Mobile view:
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/b1d775ed-00c9-4f6b-aa49-3cc7e69f68fb)

Reply:
Should be applied globally!